### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 5tran-alexil**
Style/grammar edits
Closes T-773857

### CODE REVIEW

**Comment by 5tran-alexil** [comment]
@ediril 
Convex source link throws a 404
Screenshot 2024-08-27 at 23 46 34

**Review by ediril — APPROVED** [review]

**Comment by 5tran-alexil** [comment]
@fivetran/menon please take a look at the Convex link throwing a 404.

**Review by varundhall — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
 
-> To run an example connector or destination from `examples` directory, follow the Readme specific to the example. 
+> To run an example connector or destination from `examples` directory, follow the README specific to the example. 
 
 ### Other Examples
-You can look at the following public repos to see how other partner's have built their connectors:
+You can look at the following public repos to see how other partners have built their connectors:
 #### Connectors
 * [PlanetScale](https://github.com/planetscale/fivetran-source)
 * [Convex](https://github.com/get-convex/convex-fivetran-source)
```
Comment (varundhall) [inline README.md:24]: @5tran-alexil this link is outdated
here is the correct link https://github.com/get-convex/convex-backend/tree/main/crates/fivetran_source

Comment (varundhall) [reply README.md]: I have updated it.

**Pull request merged**